app/changelog/chart-operator.yaml

```diff
@@ -1,3 +1,13 @@
+- version: 0.12.0
```

Comment:
This is sad since the format of the current changelog could conform only one item even though it should contain multiple. This is the part from `versionbundle` legacy.

Reply:
Consider extracting these additions to the changelog into a separate PR, that will make this one smaller + other release definitions can make use of the app release definitions, and platform release definitions can be tested in parallel

@rossf7 you were interested in have 0.12.0 in azure 11.2.1, correct?

Reply:
Sure I created a different PR for this. https://github.com/giantswarm/releases/pull/153